less/responsive-visibility.less

```diff
@@ -12,7 +12,7 @@
 // --------------------------------------------------------------------------
 
 .visible--palm {
-    display: block;
+    display: flex;
```

Comment:
Be aware that these classes are used for overall responsive visibility, not only in connection to grid. We should either prefix them (`e-`?) or create different classnames to affect visibility that is not related to the grid itself.